### DISCUSSION TITLE
ZydisEncoderEncodeInstruction How to set runtime-address

### DISCUSSION THREAD

**Discussion by injertao** [zyantific/zydis#359] [Q&A]
The code I want to get is based on runtime address.
Zydis + keystone can meet my needs, but it will make the volume larger
thanks.

**Comment by flobernd** [comment]
@injertao
Could you please give a short code example or a more detailed explaination of what you are going to achieve?

**Comment by injertao** [comment]
More than 1000 bytes of code, x64, which contains a variety of codes
